index.html

```diff
@@ -5,31 +5,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Henry's Portfolio</title>
     <link rel="stylesheet" href="./styles.css" />
+
+    <!-- Fontawesome CDN -->
```

Comment:
- [OPTIONAL] Your code on the repository has comments. As a good practice to ensure a cleaner code all comments must be removed from the hosted version of the code on the repository.

Reply:
You guys are great! I will resolve this right now, push it and merge.

I am very grateful.